.travis.yml

```diff
@@ -18,7 +18,6 @@ git:
   submodules: false
 
 before_install:
-  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
   - git submodule update --init --recursive
```

Comment:
Can this be removed as well? Including remvoving the `submodules: false` on the top?